src/amitaRJun20/MeetingRoom.java

```diff
@@ -0,0 +1,132 @@
+/*Assignment 27 : 14th July 2020
+Program to demonstrate overloading behavior.
+Create a class MeetingRoom having method bookMeetingRoom which should provide
+below facility:
+meeting room names: Nalanda, Takshashila, AgraFort, and PratapGadh
+a. If any user call method bookMeetingRoom() at that time�Nalanda� meeting room will be booked for 1 hr.
+b. Write a method that helps to book a meeting room other than "Nalanda" for default timing would be 1 hr.
+c. Write a method which helps to book a meeting room by giving the name of the meeting room and given hrs.
+d. displayInfo() method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs].
+e. displayAvilableMeetingRoomForBooking() method will print available meeting room names. 
+     [i.e. If Nalanda and Takshashila is booked then this method should print "Available meeting rooms: AgraFort and 
+     PratapGadh"] */
+
+package amitaRJun20;
+
+public class MeetingRoom {
+	static boolean nalandaBookingStatus, takshashilaBookingStatus, agraFortBookingStatus, pratapGadhBookingStatus;
+	static double nalandaBookingHours, takshashilaBookingHours, agraFortBookingHours, pratapGadhBookingHours;
+
+	enum MEETINGROOM {
+		NALANDA, TAKSHASHILA, AGRAFORT, PRATAPGADH;
+	}
+
+	void bookMeetingRoom() {
```

Comment:
Hi Amita, 
You can directly call bookMeetingRoom(MEETINGROOM meetingRoomName, double hours) method there as we have already written this condition that method